main/tests/server/src/com/google/refine/model/recon/StandardReconConfigTests.java

```diff
@@ -164,6 +165,27 @@ public void testReconstructNoType() throws IOException {
         assertNull(config.typeName);
     }
 
+    @Test
+    public void testGetBatchSize() throws IOException {
+
+        String json = "{\"mode\":\"standard-service\","
+                + "\"service\":\"https://tools.wmflabs.org/openrefine-wikidata/en/api\","
+                + "\"identifierSpace\":\"http://www.wikidata.org/entity/\","
+                + "\"schemaSpace\":\"http://www.wikidata.org/prop/direct/\","
+                + "\"type\":null,"
+                + "\"autoMatch\":true,"
+                + "\"batchSize\":50,"
+                + "\"columnDetails\":["
+                + "    {\"column\":\"_ - id\","
+                + "     \"property\":{\"id\":\"P3153\",\"name\":\"Crossref funder ID\"}}"
+                + "],"
+                + "\"limit\":0}";
+        StandardReconConfig c = StandardReconConfig.reconstruct(json);
+        assertEquals(c.getBatchSize(10), 10);
+        assertEquals(c.getBatchSize(1200), 12);
```

Comment:
I am asking myself if for a project with 1200 rows and a service that supports batches of 50 queries, we really want to use a batch size of 12.
I think it's not so crucial that the progress report is granular down to 1%. Personally I am okay to accept progress jumps of up to 10%  if it helps speed up the entire process. I'll ask on the issue if people would agree with using ` / 10` instead of ` / 100`.

Reply:
Yes,I have personally tested this on a data set and 12 is very slow!